source/preprocessor.tex

```diff
@@ -1387,6 +1387,7 @@
 \indextext{macro!pragma operator}%
 \indextext{operator!pragma|see{macro, pragma operator}}
 
+\pnum
```

Comment:
Also move the example into the same paragraph, otherwise we don't really have just "one paragraph".